package.json

```diff
@@ -2,7 +2,7 @@
   "name": "root",
   "private": true,
   "scripts": {
-    "build": "lerna run build",
+    "build": "cd packages/snowpack && yarn build",
```

Comment:
In the `test.yml` GitHub Action, Lerna needs another `yarn --force` to symlink the binstub into the packages.

With `lerna run build`, we were building all the templates; however, we didn’t need to. We actually do this in the template test. `snowpack` is the only package we need to build, so this changes that.